build-config/resources/Ballerina.toml

```diff
@@ -7,7 +7,7 @@ keywords = ["security", "hash", "hmac", "sign", "encrypt", "decrypt", "private k
 repository = "https://github.com/ballerina-platform/module-ballerina-crypto"
 icon = "icon.png"
 license = ["Apache-2.0"]
-distribution = "2201.11.0"
+distribution = "2201.12.0"
```

Comment:
Have to change this on the Ballerina.toml, right?

Reply:
That will be changed automatically during the build right?